SDGupdater/R/get_modal_values.R

```diff
@@ -0,0 +1,23 @@
+#' Get modal values
+#'
+#' Finds which value occurs most frequently. If there is more than one mode, all
+#' will be returned
+#'
+#' @seealso From thread:
+#'   https://stackoverflow.com/questions/2547402/how-to-find-the-statistical-mode
+#'
+#' @param values Numeric or character vector.
+#'
+#' @return Vector of same type as values.
+#'
+#' @examples
+#' get_modal_values(1,2,3,1,3)
```

Comment:
Lines 14, 15, and 16 for the examples, the numbers should be fed as a vector, so `get_modal_values(c(1,2,3,1,3))`